modules/server/search.inc.php

```diff
@@ -13,7 +13,7 @@
 $ms2->AddSelect('u.userid');
 $ms2->AddResultField(t('Name'), 's.caption');
 $ms2->AddResultField(t('Servertyp'), 's.type', 'ServerType');
-$ms2->AddResultField(t('IP-Adresse / Domain'), 's.ip');
+$ms2->AddResultField(t('IP-Adresse / Domain'), 'INET6_NTOA(s.ip) AS ip');
```

Comment:
Shouldn't be this `realip` then? Or doesn't it have a direct relation to the other usages?

Reply:
Well... I initially fixed this code (overview page) and then noticed that the details view was still broken.
I didn't want to go far from "ip" here because that's how that field was always called.
However, in _show_details.php_  a select * is done which also returns "a.ip" as "ip". Now if I would have called the INET6_NTOA result "ip" as well that query would return both columns as "ip" ... which would become rather interesting I guess; at least confusing. I then looked at the amount of columns of the _server_ table and ended up with calling that field _realip_.
Reconsidering this, I guess the best way would be me changing this in _show_details.php_ from a SELECT * to specifying all the columns and also calling the resulting field "ip" there.
What do you think?

Reply:
> Reconsidering this, I guess the best way would be me changing this in show_details.php from a SELECT * to specifying all the columns and also calling the resulting field "ip" there.

Yes. I am for this. Personally, I like to avoid `SELECT *` and to make it explicit which fields you are requesting. Often this makes things easier. Do you want to adjust it in this PR or in a new one?

Reply:
Will do it here.